client/src/components/CommentInlineEditor.vue

```diff
@@ -50,4 +67,20 @@ const onSaveMessage = () => emit("saveMessage", message.value);
 const saveButtonIsDisabled = computed(() => message.value.length < 3);
 
 const onCancel = () => emit("closeEditor");
+// Emoji Picker
+const ta = ref(null);
+const handleEmojiClick = (detail) => {
+  if (!detail.unicode) return;
```

Comment:
Die emojis werden zwar in der Textarea angezeigt. Allerdings wenn ich auf Senden/Speichern klicke, sind diese nicht in der Message enthalten.

Reply:
fixed